geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommand.java

```diff
@@ -216,18 +219,76 @@ public ResultModel createRegion(
               new Object[] {regionPath}));
     }
 
+    RegionConfig regionConfig = new RegionConfig();
+
+    // get the initial set of attributes either from shortcut or from the template region
+    InternalConfigurationPersistenceService persistenceService =
+        getConfigurationPersistenceService();
+    if (regionShortcut != null) {
+      regionConfig.setType(regionShortcut.name());
+    }
+    // if cluster config is enabled, find the template configuration from cluster config
+    else if (persistenceService != null) {
+      RegionConfig templateRegionConfig = null;
+      for (String group : persistenceService.getGroups()) {
+        CacheConfig cacheConfig = persistenceService.getCacheConfig(group, true);
+        templateRegionConfig =
+            CacheElement.findElement(cacheConfig.getRegions(), templateRegion.substring(1));
+        if (templateRegionConfig != null) {
```

Comment:
A corner case here is if we have a mix of **accessor** and **hosting** members - the template would look different on different members. In this case, I suggest we fail and not allow that.

Reply:
good find. Will add in checks.

Reply:
fixed in latest commit